wls-wahlvorbereitung-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/service/urnenwahlschliessungsuhrzeit/UrnenwahlSchliessungsUhrzeitService.java

```diff
@@ -0,0 +1,70 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.service.urnenwahlschliessungsuhrzeit;
+
+import de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.domain.UrnenwahlSchliessungsUhrzeitRepository;
+import de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.exception.ExceptionFactory;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.slf4j.MDC;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.parameters.P;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class UrnenwahlSchliessungsUhrzeitService {
+
+    private final UrnenwahlSchliessungsUhrzeitRepository urnenwahlSchliessungsUhrzeitRepository;
+    private final UrnenwahlSchliessungsUhrzeitModelMapper urnenwahlSchliessungsUhrzeitModelMapper;
+    private final UrnenwahlSchliessungsUhrzeitValidator urnenwahlSchliessungsUhrzeitValidator;
+    private final ExceptionFactory exceptionFactory;
+
+    @PreAuthorize(
+        "hasAuthority('Wahlvorbereitung_BUSINESSACTION_GetUrnenwahlSchliessungsUhrzeit')"
+                + "and @bezirkIdPermisionEvaluator.tokenUserBezirkIdMatches(#wahlbezirkID, authentication)"
+    )
+    public Optional<UrnenwahlSchliessungsUhrzeitModel> getUrnenwahlSchliessungsUhrzeit(@P("wahlbezirkID") final String wahlbezirkID) {
+        log.debug("#getUrnenwahlSchliessungsUhrzeit");
+        log.debug("in: wahlbezirkID > {}", wahlbezirkID);
+
+        urnenwahlSchliessungsUhrzeitValidator.validWahlbezirkIDOrThrow(wahlbezirkID);
+
+        val dataFromRepo = urnenwahlSchliessungsUhrzeitRepository.findById(wahlbezirkID);
+
+        log.debug("out: urnenwahlSchliessungsUhrzeit > {}", dataFromRepo.orElse(null));
+
+        return dataFromRepo.map(urnenwahlSchliessungsUhrzeitModelMapper::toModel);
+    }
+
+    @PreAuthorize(
+        "hasAuthority('Wahlvorbereitung_BUSINESSACTION_PostUrnenwahlSchliessungsUhrzeit')"
```

Comment:
Uhrzeit war im alten Service mit kleiner Uhrzeit -> `Wahlvorbereitung_BUSINESSACTION_PostUrnenwahlSchliessungsUhrzeit`, sowie in dem yml-File für Keycloak.

Reply:
Resolved with a61d0170e58b8d9e79c025ecbd73f7b292b509c6